swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -526,7 +526,7 @@ public final class MenuBar: NSObject, ObservableObject {
   }
 
   func internetResourceToggleTitle() -> String {
-    store.internetResourceEnabled == true ? "Disable this resource" : "Enable this resource"
+    store.configuration?.internetResourceEnabled == true ? "Disable this resource" : "Enable this resource"
```

Comment:
```suggestion
    store.configuration?.internetResourceEnabled ? "Disable this resource" : "Enable this resource"
```

Seems odd that you need to test a boolean for `true`?

Reply:
It can be unset, which we probably want to keep state for (user has not made selection)